### PR TITLE
Node v4/v6 removal and associated tasks

### PR DESCRIPTION
**Pull request by fgalan**
- [x] Remove Node v4 and v6 from Travis
- [x] Set Node version >=8 in packages.json 
- [x] Update logops to 2.0.0
- [x] Update dependencies based in npm audit
- [x] Replace grunt by npm scripts

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,45 +16,40 @@
...
   "main": "lib/fiware-pep-steelskin",
   "engines": {
-    "node": ">=4.8.4"
+    "node": ">=8.12.0"
   },
   "scripts": {
-    "test": "grunt"
+    "clean": "rm -rf package-lock.json && rm -rf node_modules && rm -rf coverage",
+    "test": "mocha --recursive 'test/unit/**/*.js' --reporter spec --timeout 3000 --ui bdd --exit",
+    "test:tools": "mocha --recursive 'test/tools/**/*.js' --reporter spec --timeout 3000 --ui bdd --exit",
```
Comment (fgalan) [inline package.json]: Gruntfile used for unit test:

```
            test: {
                src: ['test/unit/**/*.js', 'test/tools/**/*.js'],
                options: {
                    jshintrc: 'test/.jshintrc'
                }
},
```

So I have created one script (the default one) for test/unit and other (`test:tools`) for test/tools.

However, having said that, not sure if `test:tools` is really doing anything. I don't know what is the purpose of the code under `test/tools`. Maybe the  `test:tools` script can be removed? Maybe even the code under test/tools can be removed?

**Comment by fgalan** [comment]
After last changes `npm install` shows:

```
added 236 packages from 739 contributors and audited 579 packages in 16.527s
found 0 vulnerabilities
```

**Comment by fgalan** [comment]
@cesarjorgemartinez given this PR changes the grunt intervace (i.e. "grunt test" is now "npm test") please provide LGTM to ensure PR is ready for merge.

File: package.json (hunk 1 above)
Comment (fgalan) [reply package.json]: `npm test` result:

```
257 passing (20s)
```

Comment (fgalan) [reply package.json]: `npm run test:tools` result:

```
0 passing (6ms)
```

Comment (fgalan) [reply package.json]: Removing tools code, then run tests:

```
$ rm test/tools/*
$ npm test

> fiware-pep-steelskin@1.4.0-next test /data/src/fiware-pep-steelskin
> mocha --recursive 'test/unit/**/*.js' --reporter spec --timeout 3000 --ui bdd --exit

module.js:550
    throw err;
    ^
```

Comment (fgalan) [reply package.json]: In sum, it seems code in test/tools is needed by test/units (so it cannot be removed) but there isn't any runable test un test/tools itself (so the tests:tools npm script can be removed).

Fixed in 9bf9922

**Comment by cesarjorgemartinez** [comment]
LGTM

**Pull request merged**